src/reports.ts

```diff
@@ -2,13 +2,12 @@ import { PaginatedResponse } from './common';
 import request from './util/request';
 
 export type GetMeetingParticipantReportsParams = {
-  page_size?: number
```

Comment:
Why was `page_size` removed? I still see it in the [API docs](https://marketplace.zoom.us/docs/api-reference/zoom-api/reports/reportmeetingparticipants).

Reply:
Apologies, oversight. Fixed